src/shared/components/query/CancerStudySelector.tsx

```diff
@@ -134,6 +134,20 @@ export default class CancerStudySelector extends React.Component<ICancerStudySel
         );
     });
 
+    @computed
+    get checkReferenceGenome() {
```

Comment:
if i understand correctly, this checks if all selected studies have same referencegenome.  It true, lets call it checkForSingleReferenceGenome

Reply:
yes, you are right. Will change.

Reply:
renamed to checkForSingleReferenceGenome